inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+example projects, we have added some annotations already as examples. If you export a project (see  xref:export_in_getting_started[Export]) and import it again (as we just did with the example project in <<Download and Import an Example Project>>), there will be the same annotations like before.
+
+*Step 3 - Create an Annotation:* After opening the document, select *Named entity* from the *Layer* dropdown menu on the right side of the screen to create your first annotation. Then, use the mouse to select a word in the annotation area, e.g. _in my home_ in line one. When you release the mouse button, the annotation will immediately be created and you can edit its details in the right sidebar (see next paragraph).
+
+*_Note:_* All annotations will be saved automatically without clicking an extra save-button. This is why there is none.
+
+
+*Congratulations, you have created your first annotation!*
+
+
+Now, let‘s examine the right panel to edit the details. You find the panel named _Layer_ on top and _Annotation_ below.
+
+In the *Layer*-dropdown, you can choose the layer you want to annotate with as we just did. You always have to choose it before you make a new annotation. After an annotation has been created, its layer cannot change any more. In order to change it, you need to delete it, select the right layer and create a new annotation.
+
+If you are not sure what layers are, check xref:layers_and_features_in_getting_started[the box on _Layers And Features_ in the section Project Settings]. In order to learn how to adjust and create them for your purpose, see section <<Layers>> in the main documentation.
+
+
+In the *Annotation* panel, you see the details of a selected
+annotation. 
+
+image:getting_started_annotation_panel.png[image,width=284,height=447]
+
+It shows the layer the annotation is made in (field „Layer"; here: _Named entity_)
+and what part of the text has been annotated (field „Text“; here _in my home_). Below, you can see and modify what has been entered for each of the so called *Features*. If you are not sure what features are, check xref:layers_and_features_in_getting_started[the box on _Layers And Features_ in the section Project Settings]. (Here: The layer _Named entity_ has the features "identifier" and "value". The *identifier* tells, to which entity in the knowledge base the annotated text refers to. For example, in case the home referred to here is a location the knowledge base knows, you can choose it in the dropdown of this field. The *value* tells if it is a Location (LOC) like here, a Person (PER), Organization (ORG) or any other (OTH).) +
+You may enter free text here or work with tagsets to have a well
+defined set of lables to enter so all of the users within one project will use the same lables. You can modify and create tagsets in the project settings. See section xref:tagsets_in_getting_started[Tagsets] in Getting Started or check the main documentation for <<Tagsets>>.
```

Comment:
labels

Reply:
?

Reply:
You wrote "...to have a well
defined set of **lables**", it should be "labels"